plone/outputfilters/filters/resolveuid_and_caption.py

```diff
@@ -324,6 +324,7 @@ def unknown_starttag(self, tag, attrs):
                 scheme = urlsplit(href)[0]
                 if not scheme and not href.startswith('/') \
                         and not href.startswith('mailto<') \
```

Comment:
It was introduced here: https://github.com/julianhandl/plone.outputfilters/commit/13a901877b02f8358a5905f86b9d8b924210de6e
To me, it looks like a bug.